lnrpc/rpc.proto

```diff
@@ -2295,6 +2295,25 @@ message Payment {
 
     ///  The fee paid for this payment in milli-satoshis
     int64 fee_msat = 12 [json_name = "fee_msat"];
+
+    /// The HTLCs made in attempt to settle the payment [EXPERIMENTAL].
+    repeated HTLCAttempt htlcs = 13[json_name = "htlcs"];
```

Comment:
Should we deprecate the payment level route field now? There are other fields like `value` that can be derived from the htlc list. That is just redundant. But the payment level route is not very well defined. Users may keep using it and then run into unpleasant surprises when they start doing mpps.

Reply:
I think it may be premature to deprecate now considering that the field is still tagged EXPERIMENTAL. It is somewhat ill-defined, but it is no worse than it exists today. The route field right now will show the route from the last attempt, which is also what is shown for an MPP payment